executor/executor.go

```diff
@@ -166,13 +173,14 @@ func newBaseExecutor(ctx sessionctx.Context, schema *expression.Schema, id fmt.S
 // return a batch of rows, other than a single row in Volcano.
 // NOTE: Executors must call "chk.Reset()" before appending their results to it.
 type Executor interface {
+	This() *baseExecutor
```

Comment:
`This` can be unexported, and why not name it `base`?

Reply:
Done